test/old_tests/UnitTests/as_implements.cpp

```diff
@@ -74,6 +74,16 @@ TEST_CASE("as_implements")
     REQUIRE(foo.get() == foo2.get());
 }
 
+TEST_CASE("as_implements_uniform_initialization")
+{
+    // uniform initialization relies on IStringable(IStringable&&),
+    // which requires non-const rvalue semantics.
+    com_ptr<Foo> foo = make_self<Foo>();
+    IStringable stringable{ foo.as<IStringable>() };
```

Comment:
```suggestion
    IStringable stringable{ *foo };
```

Isn't something like this required to actually invoke the implicit conversion operator? Otherwise you're just calling the explicit `as` method provided by `com_ptr` which in turn calls QI.

Reply:
*facepalm*